docs/source/_templates/indexcontent.html

```diff
@@ -10,7 +10,10 @@
   <h1>Hyperledger Sawtooth Raft Documentation</h1>
 
   <p class="biglink"><a class="biglink" href="{{ pathto("introduction") }}">{% trans %}Introduction{% endtrans %}</a><br/>
-    <span class="linkdescr">{% trans %}learn about Hyperledger Sawtooth{% endtrans %}</span></p>
+    <span class="linkdescr">{% trans %}learn about Hyperledger Sawtooth Raft{% endtrans %}</span></p>
+
+  <p class="biglink"><a class="biglink" href="{{ pathto("configuring_deploying") }}">{% trans %}Configuring and Deploying Sawtooth Raft{% endtrans %}</a><br/>
```

Comment:
Change "Configuring and Deploying Sawtooth Raft" to "Configuration and Deployment"